views/build/.htaccess

```diff
@@ -0,0 +1,4 @@
+<IfModule mod_rewrite.c>
```

Comment:
I don't think that `.htaccess` files are necessary for the task

Reply:
Yes, it was generated automatically by mistake